internal/payloads/sources_payload.go

```diff
@@ -7,11 +7,10 @@ import (
 	"github.com/go-chi/render"
 )
 
-type SourceID struct {
-	SourceId string `json:"source_id"`
-}
+// See clients.Source
 type SourceResponse struct {
```

Comment:
Would we be able to add the provider type here?
We apparently have consumer for that info and it might be helpful even in the UI in the future.

Reply:
I would like to either do cleanup in this PR (removal of things) or nothing. I can probably put it back and create a ticket.

But whoever is using that number, it will change between prod/ephemeral and stage likely. It is a not good data source.

Reply:
Created https://issues.redhat.com/browse/HMS-1829

This is now ready for review and merge, I want to do followup for reservations and I am done!